build.gradle

```
@@ -22,6 +22,7 @@ dependencies {
     implementation 'org.springframework.boot:spring-boot-starter-actuator'
     implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
     implementation 'org.springframework.boot:spring-boot-starter-web'
+    compile group: 'org.hibernate', name: 'hibernate-validator', version: '6.1.6.Final'
```

It would be good to use the same format as above
```suggestion
    implementation 'org.hibernate:hibernate-validator:6.1.6.Final'
```